apps/api/src/services/scrapers/scraper_manager.py

```
@@ -0,0 +1,61 @@
+
+
+class ScraperManager:
+
+    bluesky_scraper: Optional[BlueskyScraper] = None
+    weibo_scraper: Optional[WeiboScraper] = None
+    general_scraper: Optional[GeneralScraper] = None
+
+    scrapers = {"bluesky": bluesky_scraper,
+                "weibo": weibo_scraper,
+                "other": general_scraper,
+                "unknown": general_scraper}
```

Ruff (0.15.1)

[warning] Mutable default value for class attribute

(RUF012)